examples/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ add_subdirectory(c)
 # their own binary directory (../../wt). This being the directory structure produced by the autoconf
 # build. Symlink the wt binary in an expected path for backwards compatibility.
```

Comment:
Do we need to update this comment as we are doing a copy rather than a symlink?

Reply:
Good point, will fix!

Reply:
Updated :)